graphpype/tests/test_utils.py

```diff
@@ -0,0 +1,92 @@
+from graphpype.utils import (get_first, get_second, show_length, show_files,
```

Comment:
you should order the imports correctly.

They go from the most general functions to the less general ones. See pep8 documentation: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
done